infra/terraform/base_stacks/vpc.tf

```diff
@@ -9,17 +9,16 @@ resource "aws_cloudformation_stack" "vpc" {
   capabilities = ["CAPABILITY_AUTO_EXPAND", "CAPABILITY_IAM"]
 
   parameters = {
-    SSMParametersStoreEnabled = "Yes"
-    // TODO: Enable AWS Service APIs as required
-    CloudFormationEndpointEnabled = "Yes"
-    DynamoDBApiEnabled       = "Yes"
-    ExecuteApiGatewayEnabled = "Yes"
-    KMSApiEnabled            = "Yes"
-    LogsApiEnabled           = "Yes"
-    S3ApiEnabled             = "Yes"
-    SQSApiEnabled            = "Yes"
-    SecretsManagerApiEnabled = "Yes"
-    SSMApiEnabled            = "Yes"
-    AllowRules               = "pass tls $HOME_NET any -> $EXTERNAL_NET 443 (tls.sni; content:\".account.gov.uk\"; endswith; msg:\"Pass TLS to *.account.gov.uk\"; flow:established; sid:2001; rev:1;)"
+    AllowRules                      = "pass tls $HOME_NET any -> $EXTERNAL_NET 443 (tls.sni; content:\".account.gov.uk\"; endswith; msg:\"Pass TLS to *.account.gov.uk\"; flow:established; sid:2001; rev:1;)"
```

Comment:
Has anything changed here apart from the order? I can't see anything 👀

Reply:
No, in this PR it's just the order - it was all a bit messy...